Project.toml

```diff
@@ -1,15 +1,15 @@
 name = "UnitfulAstro"
 uuid = "f2a1aa4e-5634-4978-a6f6-8c9e368f14ef"
 license = "MIT"
-version = "0.2.0"
+version = "0.3.0"
```

Comment:
The version number should be incremented only on the commit that will mark a new release. Is this the case?

Reply:
The problem is that currently if you install `UnitfulAstro 0.2.0` you get the release version which does not contain the magnitudes and other changes since November. You have to install `UnitfulAstro#master` which is still 0.2.0 but has different functionality. It would be nice to have at least `0.2.1` such that someone can pin UnitfulAstro to use the magnitudes. 

This came up because I want to use the magnitudes in some of the calculations for `Spectra.jl`